velox/exec/Window.h

```diff
@@ -22,6 +22,14 @@
 
 namespace facebook::velox::exec {
 
+// Pointer to the function used to compute window frame bounds.
+typedef vector_size_t (*windowFrameFunctionPtr)(
```

Comment:
I think W needs to be in capital here since this is the name of a type and not a value.